dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java

```diff
@@ -70,6 +70,7 @@
 import org.apache.cxf.transport.AbstractMultiplexDestination;
 import org.apache.cxf.transport.Assertor;
 import org.apache.cxf.transport.Conduit;
+import org.apache.cxf.transport.http.AbstractHTTPDestination.BackChannelConduit;
```

Comment:
why is this necessary?

Reply:
It is an internal class that eclipse automatically adds for some reason... I'll remove it.